### PR TITLE
Airflow Dockerfile

### PR DESCRIPTION
**Pull request by eanylin**
Dockerfiles for Airflow

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/dockerfiles/96)

### CODE REVIEW

File: airflow/config/airflow.cfg [hunk 1]
```diff
@@ -0,0 +1,214 @@
+[core]
```
Comment (alanmeadows) [inline airflow/config/airflow.cfg:1]: This is a fine default, obviously we will want to override this in the chart.  Will comment there on how.

File: airflow/script/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+CMD="airflow"
```
Comment (alanmeadows) [inline airflow/script/entrypoint.sh:3]: Ideally, we simply extract these from airflow.cfg, rather then needing to define them both in the config and as environment variables to the container. Requires a little chopping voodoo on the strings in the cfg as they are not in this format. Also, it looks like these are hard coded, and that if given over the environment (via k8s pod) they would be ignored.

**Review by alanmeadows — CHANGES_REQUESTED** [review]

File: airflow/script/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+POSTGRES_PORT="5432"
+RABBITMQ_HOST="rabbitmq"
+RABBITMQ_CREDS="airflow:airflow"
+FERNET_KEY=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print FERNET_KEY")
+
+# Generate Fernet key
+sed -i "s/{FERNET_KEY}/${FERNET_KEY}/" $AIRFLOW_HOME/airflow.cfg
+
+# wait for rabbitmq
+if [ "$1" = "webserver" ] || [ "$1" = "worker" ] || [ "$1" = "scheduler" ] || [ "$1" = "flower" ] ; then
```
Comment (larryrensing) [inline airflow/script/entrypoint.sh]: unless i'm mistaken -- you should be able to remove these lines and instead leverage a kubernetes entrypoint to wait until postgres and rabbit have endpoints within kubernetes

Comment (alanmeadows) [reply airflow/script/entrypoint.sh]: agreed

Comment (eanylin) [reply airflow/script/entrypoint.sh]: Agree.  Will leverage on kubernetes entrypoint.

**Pull request merged**